contracts/lease/src/api/open/mod.rs

```diff
@@ -108,6 +110,28 @@ impl InterestPaymentSpec {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone)]
```

Comment:
we'll need `PartialEq, Eq, JsonSchema` as well once we define a `position` attribute of the `NewLeaseForm`